Makefile

```diff
@@ -101,16 +64,24 @@ LDLIBS  += -lm -lgcc -lc
 
 include $(BOLOS_SDK)/Makefile.glyphs
 
-APP_SOURCE_PATH += src
-SDK_SOURCE_PATH += lib_stusb lib_stusb_impl
+include $(BOLOS_SDK)/lib_standard_app/Makefile.standard_app
 
-ifneq ($(TARGET_NAME),TARGET_STAX)
-SDK_SOURCE_PATH += lib_ux
-endif
+# Here custom flags can be added
+CUSTOM_APP_FLAGS = 0x000 # No custom flags needed, see appflags.h in SDK for flags definition
+APP_FLAGS = $(shell echo $$(( $(STANDARD_APP_FLAGS) + $(CUSTOM_APP_FLAGS) )) )
 
-ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_NANOX TARGET_STAX))
-    SDK_SOURCE_PATH += lib_blewbxx lib_blewbxx_impl
-endif
+APP_LOAD_PARAMS += --appFlags $(APP_FLAGS)
+
+APP_LOAD_PARAMS += --curve secp256k1
+
+APP_LOAD_PARAMS += --path "44'"
+
+APP_LOAD_PARAMS += $(STANDARD_LOAD_PARAMS)
+
+DEFINES += $(STANDARD_APP_DEFINES)
+
+APP_SOURCE_PATH += src
+SDK_SOURCE_PATH += $(STANDARD_SDK_SOURCE_PATH)
 
 load: all
```

Comment:
Going even further, what do you think of putting the makefile target load / load-offline / delete / listvariants in your too?